src/plaster_yaml/__init__.py

```diff
@@ -1,5 +1,15 @@
-import pkg_resources
+import sys
+
+if sys.version_info < (3, 10):
+    import importlib_metadata
+
+    class importlib:
+        pass
+
+    setattr(importlib, "metadata", importlib_metadata)
+else:
+    import importlib.metadata
```

Comment:
this is why I dislike that. 11 lines  of code to retrieve a version number. this is outstanding.

please remove python 3.7 support and everything will be much simpler.